docs/tilemap.asciidoc

```diff
@@ -25,7 +25,10 @@ Before you choose a buckets aggregation, specify if you are splitting the chart
 Coordinates* on a single chart. A multiple chart split must run before any other aggregations.
 
 Tile maps use the *Geohash* aggregation as their initial aggregation. Select a field, typically coordinates, from the
-drop-down. The *Precision* slider determines the granularity of the results displayed on the map. 
+drop-down. The *Precision* slider determines the granularity of the results displayed on the map. See the documentation 
+for the {ref}/search-aggregations-bucket-geohashgrid-aggregation.html#_cell_dimensions_at_the_equator[geohash grid] 
+aggregation for details on the area specified by each precision level. As of the 4.1 release, Kibana supports a maximum 
+geohash length of 7.
```

Comment:
Added this sentence:

"NOTE: Higher precisions increase memory usage for the browser displaying Kibana as well as for the underlying Elasticsearch cluster."

Also added 

"NOTE: By default, the *Change precision on map zoom* box is checked. Uncheck the box to disable this behavior."

after the Geohash entry. Question: Why is this option here, and not in Options with the other ones?